packages/@aws-cdk-testing/framework-integ/test/aws-dynamodb/test/integ.dynamodb-v2.policy.ts

```diff
@@ -31,6 +31,9 @@ class TestStack extends Stack {
     });
 
     table.grantReadData(new iam.AccountPrincipal('123456789012'));
+    table.addReplica({
+      region: 'eu-west-2',
+    });
```

Comment:
This test didn't even set the feature flag, so my understanding is that it should use the table's resource policy but doesn't seem to be the case in the output template.json. May I ask how you ran the tests locally and update the snapshot?